src/shared/warn-once.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
```

Comment:
This file has been moved from `src/sidebar/util/warn-once.js` and modified slightly to make it a better drop-in replacement for `console.warn`.